pysat/_files.py

```diff
@@ -269,35 +271,34 @@ def _load(self, prev_version=False):
         else:
             return pds.Series([], dtype='a')
 
-
     def refresh(self):
         """Update list of files, if there are changes.
-        
+
         Calls underlying list_rtn for the particular science instrument.
         Typically, these routines search in the pysat provided path,
         pysat_data_dir/platform/name/tag/,
         where pysat_data_dir is set by pysat.utils.set_data_dir(path=path).
-        
+
 
         """
 
         output_str = '{platform} {name} {tag} {sat_id}'
         output_str = output_str.format(platform=self._sat.platform,
-                                       name=self._sat.name, tag=self._sat.tag, 
+                                       name=self._sat.name, tag=self._sat.tag,
                                        sat_id=self._sat.sat_id)
         output_str = " ".join(("pysat is searching for", output_str, "files."))
         output_str = " ".join(output_str.split())
-        print (output_str)
-        
+        print(output_str)
+
         info = self._sat._list_rtn(tag=self._sat.tag, sat_id=self._sat.sat_id,
                                    data_path=self.data_path,
                                    format_str=self.file_format)
 
         if not info.empty:
             print('Found {ll:d} of them.'.format(ll=len(info)))
         else:
-            estr = "Unable to find any files that match the supplied template. "
-            estr += "If you have the necessary files please check pysat "
+            estr = "Unable to find any files that match the supplied template."
+            estr += " If you have the necessary files please check pysat "
```

Comment:
Seems a bit weird to have the space at the front of the sentence. Did the pep8 checker suggest this?

Reply:
I think this was me trying to get to below 80 characters.  The checker just points out violations, doesn't suggest corrections

Reply:
fixed.